cpp/unreal_plugins/CoreUtils/Source/CoreUtils/Config.cpp

```diff
@@ -12,6 +12,7 @@
 #include "CoreUtils/Unreal.h"
 #include "CoreUtils/YamlCpp.h"
 
+bool Config::is_enabled_;
```

Comment:
Update: based on an offline design discussion, we agreed that it is better to go with the sketch as it is proposed in this PR. But let's use `bool Config::s_is_enabled_` to match our existing convention for static variables.